kubernetes/hmda-analytics/templates/config-maps.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ .Release.Name }}-analytics-config"
+data:
+  postgres.ssl: "{{ .Values.postgresql.ssl }}"
```

Comment:
You will need to add these four values to the values file so that the config map can pick them up.

Reply:
addressed in new commits.